builds/Makefile

```diff
@@ -862,7 +866,7 @@ $(IOS_DESTDIR)$(XAMARIN_IPHONEOS_SDK)/usr/lib/%: .stamp-$(MONO_BUILD_MODE) $(X86
 	$(Q_LIPO) lipo $(MONO_IOS_SDK_DESTDIR)/ios-libs/ios/$(notdir $@) $(X86_64_SLICE) -create -output $@
 	$(Q) if test -d $(MONO_IOS_SDK_DESTDIR)/ios-libs/ios/$(notdir $@).dSYM; then $(CP) -R $(MONO_IOS_SDK_DESTDIR)/ios-libs/ios/$(notdir $@).dSYM $(dir $@); fi
 
-$(IOS_DESTDIR)$(XAMARIN_IPHONEOS_SDK)/Frameworks/Mono.framework/Mono: .stamp-$(MONO_BUILD_MODE) | $(IOS_DESTDIR)$(XAMARIN_IPHONEOS_SDK)/Frameworks
+$(IOS_DESTDIR)$(XAMARIN_IPHONEOS_SDK)/Frameworks/Mono.framework/Mono: .stamp-$(MONO_BUILD_MODE) $(X64_64_SLICE) | $(IOS_DESTDIR)$(XAMARIN_IPHONEOS_SDK)/Frameworks
```

Comment:
I don't think this works, because the suffix is empty in this rule. Maybe use the full variable instead (`$(X86_64_SLICE_DYLIB`), as well as below in the lipo statement as well.

Reply:
Empty suffix should be the right thing here. It didn't work because I made a typo: `s/X64_64_SLICE/X86_64_SLICE/`

```console
$ rm ./_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono
$ rm builds/x86-64-slice
$ make -C builds /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono V=1
cp /Users/lewurm/work/xi-2019-08/xamarin-macios/builds/x86-64-slice.dylib /Users/lewurm/work/xi-2019-08/xamarin-macios/builds/x86-64-slice
cp -c -R /Users/lewurm/work/xi-2019-08/xamarin-macios/external/mono/sdks/out/ios-frameworks/ios/Mono.framework /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks
cp -c -R /Users/lewurm/work/xi-2019-08/xamarin-macios/external/mono/sdks/out/ios-frameworks/ios/Mono.framework.dSYM /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks
mv /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono.tmp
lipo /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono.tmp /Users/lewurm/work/xi-2019-08/xamarin-macios/builds/x86-64-slice -create -output /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono
rm /Users/lewurm/work/xi-2019-08/xamarin-macios/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/SDKs/MonoTouch.iphoneos.sdk/Frameworks/Mono.framework/Mono.tmp
```

Reply:
hmm, looks like you were partially right: `$(suffix $@)` doesn't work when used in a dependency for a target. For some reason this only is a problem when `MONO_BUILD_FROM_SOURCE=1` is _not_ set.